add step-by-step tutorial to support new Getting Started page in IQX

derivation opened the pull request:
<!--
⚠️ If you do not respect this template, your pull request will be closed.
⚠️ Your pull request title should be short detailed and understandable for all.
⚠️ Also, please add it in the CHANGELOG file under Unreleased section.
⚠️ If your pull request fixes an open issue, please link to the issue.

✅ I have added the tests to cover my changes.
✅ I have updated the documentation accordingly.
✅ I have read the CONTRIBUTING document.
-->

### Summary
As part of the new Getting Started page being designed for IBM Q Experience by Jules Murphy and myself, we are adding a step-by-step intro for IBM Q Experience users.

closes #848

### Details and comments
This new tutorial is based on https://qiskit.org/documentation/getting_started.html 

The step-by-step is distinct from [1_start_here](https://github.com/Qiskit/qiskit-iqx-tutorials/blob/master/qiskit/1_start_here.ipynb) and [1_getting_started_with_qiskit](https://github.com/Qiskit/qiskit-iqx-tutorials/blob/master/qiskit/fundamentals/1_getting_started_with_qiskit.ipynb). It will not replace these two existing tutorials.

nonhermitian commented:
This is missing a TOC like the other fundamental notebooks have.  More importantly though, how does one square this notebook with part 1?  They seem very similar in spirit, and in name.  Does it not make more sense to go and rework part 1 in the spirit of the notebook presented here?

derivation commented:
@nonhermitian I added a table of contents and fixed the header image. 

I recommend keeping parts 0 and 1 for now, and opening an issue to square them in the future. 

Part 0 is mainly to support the new Getting Started page project, which has a lot of other work involved and a short deadline. Also, there's a substantive progression from part 0 to part 1 with new, useful info in part 1 that would overwhelm a novice had they not first seen part 0.
